src/PrestaShopBundle/Resources/config/routing/admin/improve/design/_design.yml

```diff
@@ -0,0 +1,3 @@
+_theme_catalog:
+    resource: "theme_catalog.yml"
+    prefix: /themes-catalog/
```

Comment:
Missing new line

Reply:
:/

missing CS validator configuration and build in travis.

This is the last time I fix a coding style issue, I don't have time for this /c  @eternoendless